spec/restaurant_spec.rb

```diff
@@ -0,0 +1,99 @@
+describe Restaurant do
+  let(:test_order)   { [{ napoli: 13 }, { diavola: 14 }] }
+  let(:order_output) { ['napoli: £13.00', 'Diavola: £14.00', 'Total: £27.00'] }
+  let(:menu)         { Menu::PIZZA }
+
+  let(:menu_output) do [
+    'Marinara: £8.00',
+    'Margherita: £10.00',
+    'Bufalina: £13.00',
+    'Diavola: £14.00',
+    'Napoli: £14.00',
+    'Calzone: £14.00',
+    'Fiorentina: £14.00',
+    'Capricciosa: £15.00'
+   ]
+  end
+
+  let(:customer_service) do instance_double(
+    CustomerService, :customer_service,
+    confirmation_text: 'text sent'
+  )
+  end
+
+  let(:display) do instance_double(
+    Display, :display,
+    view_menu: menu_output,
+    view_order: order_output,
+    confirm_order: nil,
+    confirm_added: nil,
+    invalid: Display::INVALID_MEAL_MESSAGE
+  )
+  end
+
+  subject do described_class.new(
+    menu: menu,
+    display: display,
+    customer_service: customer_service
+  )
+  end
+
+  describe '#view_menu' do
+    it 'displays the meals and prices in nice format' do
+      expect(subject.view_menu).to eq menu_output
+    end
+  end
+
+  describe '#add' do
+    it 'adds meal to order' do
+      subject.add(:marinara)
```

Comment:
this isn't testing whether your downcase.to_sym works, since you're passing the argument as a downcase symbol anyway.

Reply:
Yeah you're right